### DISCUSSION TITLE
I want to ask for a simple demo of nuxt. I think the demo is the easiest for novices to get started.

### DISCUSSION THREAD

**Discussion by duoluodexiaoxiaoyuan** [nuxt/nuxt#29562] [General]
Thanks for everyone's replies

**Comment by xjccc** [comment]
You could see this. https://nuxt.com/templates

**Reply by duoluodexiaoxiaoyuan** [reply]
> You could see this. https://nuxt.com/templates

thank you, I haven't seen it here before
